src/thrift.js

```diff
@@ -94,14 +94,20 @@ export default class ThriftUtils {
     static spanRefsToThriftRefs(refs: Array<Reference>): Array<any> {
         let thriftRefs = [];
         for (let i = 0; i < refs.length; i++) {
-            let refEnum;
             let ref = refs[i];
+            if (!ref) {
```

Comment:
why would this ever be the case?  These are refs from the span, we should prevent adding nulls to that array.